_episodes_rmd/04-assumptionsAndFit.Rmd

```diff
@@ -168,7 +167,7 @@ without the interaction using the tools discussed in this episode.
 > >               plot.points = TRUE, interval = TRUE,
 > >               point.size = 0.7)
 > > ```
-> > 
+> > There is little evidence from the plot that the effect of height on weight is different between sexes meaning that an interaction term is not needed. But we could assess the model fit with and without the interaction to check.

```

Comment:
This would then also need to be changed to relate to separate intercepts rather than an interaction.

Reply:
yep.